pkg/service/v1alpha1/controller.go

```diff
@@ -124,19 +121,23 @@ func (cs *controller) CreateVolume(
 	if err == nil && cvc != nil && cvc.DeletionTimestamp == nil {
 		goto createVolumeResponse
 	}
+
 	err = utils.ProvisionVolume(size, volName, rCount,
 		cspcName, snapshotID,
-		nodeID, policyName)
+		nodeTopology, policyName)
```

Comment:
name it nodeID only, as we get preferred node name. Rename nodeTopology to nodeID.

Reply:
ok , done